iOS/MasterFeed/Cell/MasterFeedTableViewSectionHeader.swift

```diff
@@ -139,7 +139,6 @@ private extension MasterFeedTableViewSectionHeader {
 	}
 	
 	func commonInit() {
-		addSubviewAtInit(unreadCountView)
```

Comment:
I'm assuming the rest of the clean up happens in another commit?

Reply:
e4cb868

Reply:
Thanks!